File: extjsapp/app/model/Record.js
Ext.define('ExtJsApp.model.Record', {
    extend: 'ExtJsApp.model.Base',

    fields: [
        { name: 'id', type: 'string' },
        { name: 'name', type: 'string' },
        { name: 'dateOfBirth', type: 'date', dateFormat: 'C' },
        { name: 'phone', type: 'string' }
    ],

    proxy: {
        type: 'rest',
        url : 'https://localhost:7268/api/records',
        cors: true,
        useDefaultXhrHeader: false
    }
});

File: webapi/Models/Record.cs
using System.ComponentModel.DataAnnotations;

namespace webapi.Models
{
    public class Record
    {
        [Key]
        public Guid Id { get; set; }
        public string Name { get; set; }
        public DateTime DateOfBirth { get; set; }
        public string Phone { get; set; }
    }
}
